docs/conceptual/runtime/index.md

```diff
@@ -2,6 +2,32 @@
 title: Runtime Overview
 ---
 
-What is the Substrate Runtime?
+## Substrate Runtime
 
+The runtime is the most critical component in your blockchain. 
+It contains the business logic that will define your blockchain's behavior,
+a.k.a. the state transition function.
 
+The Substrate runtime is composed of a set of modules (called pallets) and 
```

Comment:
This is how runtimes are written when using frame, but they can also be written without frame. Might be good to discuss what a state transition function is and emphasize that it is not block authoring, or finality, or networking. Maybe include a diagram showing that the runtime is one part of the node. And probably worth mentioning that the runtime must compile to wasm.

Reply:
I previously mentioned about the possibility of building a runtime without frame. I ended up removing it because we don't have any example so far. I can add it back to empathize the flexibility. I'll address the other points.

Reply:
Good idea. I agree, we shouldn't write it without an example to back it up. I created an issue for a recipe but we shouldn't block this pr on it. https://github.com/substrate-developer-hub/recipes/issues/131.